Support/Testing/Travis/after_success.sh

```diff
@@ -11,6 +11,10 @@
 
 set -eu
 
+if [[ "${TARGET-}" = "Darwin-X86_64" ]]; then
```

Comment:
Doesn't really matter here, but you could do `if [[ "${TARGET-}" = "Darwin-* ]] ...`.